src/Schema/Events/ImpressionTracking.ts

```diff
@@ -259,3 +259,28 @@ export interface EditProfileModalViewed {
   inquiry_id: string
   platform: Platform
 }
+
+/**
+ * User sees the banner in the app
+ *
+ * This schema describes events sent to Segment from [[BannerViewed]].
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "bannerViewed",
+ *    context_screen: "home",
+ *    context_module: "paymentFailed",
+ *    item_type: "order" | "orders"
+ *    item_id: "order-id" | ""
```

Comment:
And this `item-id` is order id with failed payment in case of a single failing payment or blank if there were multiple failed payments and we are just linking to orders history page. Makes sense?